R/fcm.R

```diff
@@ -184,9 +184,7 @@ fcm.tokenizedTexts <- function(x, context = c("document", "window"),
                                           dims = c(lengthToken , lengthToken))
         diag(result) <- 0
         result <- result + diagCount
-        
-        # order the features alphabetically
-        result <- result[order(rownames(result)), order(colnames(result))]
+        result <- result[rownames(result), colnames(result)]
```

Comment:
This simply selects all rows and columns, and is therefore redundant. Better to comment out original line 189 to show that it was considered but disabled.